src/lib/convertId.ts

```diff
@@ -20,7 +20,7 @@ async function getIdDict(update = false): Promise<{ [key: string]: string }> {
       'package',
       dictUrl
     );
-    return fs.readJsonSync(convertJson);
+    return dictUrl ? fs.readJsonSync(convertJson) : {};
```

Comment:
Do you mean convertJson?
```suggestion
    return convertJson ? fs.readJsonSync(convertJson) : {};
```